Controllers/FileManager.cs

```diff
@@ -35,9 +34,18 @@ public void Import()
                 {
                     ReadSwed(reader, db, line);
                 }
-                else if (line.Substring(0, 15) == "\"SĄSKAITOS  (LT")
+                else if (line.Substring(0, 15) == "\"SĄSKAITOS  (LT" || line.Substring(0, 12) == "ACCOUNT  (LT" || line.Substring(0,14) == "SĄSKAITOS  (LT")
                 {
-                    ReadSEB(reader, db, line);
+                    bool quotes = false;
+                    if(line[0] == '\"')
+                    {
+                        quotes = true;
+                    }
+                    ReadSEB(reader, db, line, quotes);
+                }
+                else if (line.Substring(0, 24) == "Operacijos/Balanso Tipas" || line.Substring(0, 16) == "Transaction type")
+                {
+                    ReadLum(reader, db, line);
```

Comment:
For better readablity could be writtn in `swich case` style & put in a separate method (named, for example, MachCsvFormat or MachHeader)

Reply:
I will look into it